controllers/runner_controller.go

```diff
@@ -420,6 +421,17 @@ func (r *RunnerReconciler) newPod(runner v1alpha1.Runner) (corev1.Pod, error) {
 	// Customize the pod spec according to the runner spec
 	runnerSpec := runner.Spec
 
+	if runnerSpec.WorkVolumeClaimTemplate != nil {
+		if len(runnerSpec.WorkVolumeClaimTemplate.AccessModes) == 0 {
+			runnerSpec.WorkVolumeClaimTemplate.AccessModes = []corev1.PersistentVolumeAccessMode{
+				corev1.ReadWriteOnce,
```

Comment:
👍

Reply:
Maybe we should consider throwing if no access mode is set. Lets test what k8s does and follow that behavior.